manual/manual/tutorials/lablexamples.etex

```diff
@@ -116,9 +116,11 @@ A function taking some optional arguments must also take at least one
 non-optional argument. The criterion for deciding whether an optional
 argument has been omitted is the non-labeled application of an
 argument appearing after this optional argument in the function type.
+
 Note that if that argument is labeled, you will only be able to
-eliminate optional arguments through the special case for total
-applications.
+eliminate optional arguments by totally applying the function,
+omitting all optional arguments and omitting all labels for all
+arguments.
```

Comment:
You could maybe replace  `omitting all labels for all arguments` by `omitting all labels for all *other* arguments` or `all remaining arguments` to make it clear that optional arguments are excluded.